<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cars</class>
 <widget class="QDialog" name="cars">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Список ТС</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTableView" name="oldRepairsTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>751</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="repairsListButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>120</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(235, 155, 24);
color: rgb(255, 255, 255);
border-style: solid;
border-width: 1px;
border-radius: 4px;
border-color:rgb(41, 40, 40) ;</string>
   </property>
   <property name="text">
    <string>Список ТО</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(235, 155, 24);
color: rgb(255, 255, 255);
border-style: solid;
border-width: 1px;
border-radius: 4px;
border-color:rgb(41, 40, 40) ;</string>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>58</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(235, 155, 24);
color: rgb(255, 255, 255);
border-style: solid;
border-width: 1px;
border-radius: 4px;
border-color:rgb(41, 40, 40) ;</string>
   </property>
   <property name="text">
    <string>Новое ТО</string>
   </property>
  </widget>
  <widget class="QTableView" name="carsTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>451</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
